### DISCUSSION TITLE
How to open on MacOS?

### DISCUSSION THREAD

**Discussion by kmannumber1** [AssetRipper/AssetRipper#1267] [Q&A]
I am trying to open AssetRipper but there is nowhere to open it.

**Comment by mextex** [comment]
Hi, I also tried to open AssetRipper on macOS but could not manage to open it. The macOS-howto on the website refers to versions prior 1.0.2. AssetRipper is – as far as I have understood here https://github.com/AssetRipper/AssetRipper/discussions/1280 – not a standalone app. In which browser can I open it then? Thank you!

**Reply by xwcoco** [reply]
extract files from zip
chmod +x ./AssetRipper.GUI.Free 
then 
./AssetRipper.GUI.Free
